pkg/porter/credentials.go

```diff
@@ -194,3 +206,102 @@ func (p *Porter) GenerateCredentials(opts CredentialOptions) error {
 	}
 	return nil
 }
+
+// Validate validates the args provided Porter's credential show command
+func (o *CredentialShowOptions) Validate(args []string) error {
+	switch len(args) {
+	case 0:
+		return errors.Errorf("no credential name was specified")
+	case 1:
+		o.Name = strings.ToLower(args[0])
+	default:
+		return errors.Errorf("only one positional argument may be specified, the credential name, but multiple were received: %s", args)
+	}
+
+	format, err := printer.ParseFormat(o.RawFormat)
+	if err != nil {
+		return err
+	}
+	o.Format = format
+
+	return nil
+}
+
+// ShowCredential shows the credential set corresponding to the provided name, using
+// the provided printer.PrintOptions for display.
+func (p *Porter) ShowCredential(opts CredentialShowOptions) error {
+	credsDir, err := p.Config.GetCredentialsDir()
+	if err != nil {
+		return errors.Wrap(err, "unable to determine credentials directory")
+	}
+
+	credSet := &credentials.CredentialSet{}
```

Comment:
Can you break this up into 2 functions: GetCredential and ShowCredential? It makes it easier for testing and automating porter that way.

Reply:
Argh, you'd think I would've learned this best practice from the last credentials list PR!  Thank you for the reminder (again).  Updated.